.github/workflows/stale_pr.yml

```diff
@@ -0,0 +1,23 @@
+name: Mark and Remove Stale PRs
+on:
+  schedule:
+  - cron: '0 19 * * 1'
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+    - name: PR Check
+      uses: actions/stale@v8
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: 'This PR is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
+        close-pr-message: 'This PR has had no activity in 5 days since being marked stale. Closing this PR as a result.'
```

Comment:
nit: There is no activity on the PR in 5 days since being marked stale. Closing this PR as a result.

Reply:
Updated wording